### DISCUSSION TITLE
Milestones

### DISCUSSION THREAD

**Discussion by bnord** [deepskies/DeepDiagnostics#42] [Ideas]
1. May 1: complete package
2. April 15: alpha version ready for people to test at the retreat
3. April 1: copy in all diagnostics from other places; have concept for glue (classes, input/output)
4. Mar 20: initial meeting

**Comment by voetberg** [comment]
Clarifying questions - they can wait for the meeting if need be. Do you know what "other places" are involved? And by "glue" do you mean the infra holding those diagnostics together?

**Reply by bnord** [reply]
I think I do know the other places

our homies' repositories
SBI macke lab
CTAN SBI 
stuff from other papers that we've identified.

**Reply by bnord** [reply]
yahz. glue = infra
